source/kernel/src/comms/bbq.rs

```diff
@@ -200,6 +200,9 @@ impl GrantR {
     }
 }
 
+unsafe impl Send for GrantR {}
```

Comment:
These aren't wrong but potentially not needed, do you want to keep them in, or back them out and add in a separate pass?

Reply:
i figured it was better to keep them, but :woman_shrugging: